src/govuk/helpers/_links.scss

```diff
@@ -325,17 +331,17 @@
   }
 }
 
-/// No visited state link mixin
+/// Default link styles, without a visited state
 ///
-/// Used in cases where it is not helpful to distinguish between visited and
-/// non-visited links.
+/// Makes links use the default unvisited, hover and active colours, with no
+/// distinct visited state.
 ///
-/// For example, navigation links to pages with dynamic content like admin
-/// dashboards. The content on the page is changing all the time, so the fact
-/// that you’ve visited it before is not important.
+/// Use in cases where it is not helpful to distinguish between visited and
+/// non-visited links. [Read more about links without visited states in the
```

Comment:
There's not much 'more' to read on the Design System page unfortunately - there's only one extra contextualising example sentence. Could you add that example sentence here instead?